### DISCUSSION TITLE
Should we use the GitHub Maven repo?

### DISCUSSION THREAD

**Discussion by rbotzer** [aerospike/spring-data-aerospike#528] [Team Posts]
Hi all (@Aloren ),

The repo has a 'packages' tab that can act as a Maven repo (https://github.com/aerospike-community/spring-data-aerospike/packages?package_type=Maven). Should we use this instead of Maven Central ?

**Comment by Aloren** [comment]
Hi Ronen.
I suppose that we can publish releases to Github packages as well, but I wouldn't remove Maven Central.

**Comment by rbotzer** [comment]
Great, thank you for the feedback
